go/vt/tabletserver/config.go

```diff
@@ -128,6 +135,22 @@ var DefaultQsConfig = Config{
 	TwoPCEnable:             false,
 	TwoPCCoordinatorAddress: "",
 	TwoPCAbandonAge:         0,
+
+	EnableTxThrottler: true,
+	// These are mostly the defaults set in the defaultMaxReplicationLagModuleConfig in
+	// go/vt/throttler/max_replication_lag_module_config.go
+	TxThrottlerConfig: `target_replication_lag_sec: 2
```

Comment:
Erez, I've chatted with Sugu about this and reached the following agreement:

- we'll keep using protobuf here to stay consistent with the existing vtctl command
- independent of this pull request (PR), we can tech talk it and discuss if JSON makes more sense. In that case, we would change both places (here and the vtctl command) in a separate PR.

As I've stated in the other comment before, I expect that nobody should ever change the default configuration here. And if they do, we should instead consider to break out some options e.g. the maximum replication lag.

For other, more prominent configurations e.g. the VSchema, it's a different story. There it makes sense to use JSON because it's more popular and almost every Vitess user needs to edit it.